CHANGELOG.rst

```diff
@@ -8,6 +8,7 @@ Unreleased
 * Django 3.0, 3.1 and 3.2 support added
 * Python 3.5 and 3.6 support removed
 * Django 1.11 support removed
+* Open compare view in new tab
```

Comment:
This will break when the branch is brought up to date where the changes above are now released.